lmdb.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 3.0
-name:          lmdb
-version:       0.2.5
+name:          cardano-lmdb
```

Comment:
Shouldn't this file be named `cardano-lmdb.cabal` now?

Reply:
I think so, good point